rdm/init/Makefile

```diff
@@ -10,6 +11,9 @@ all: $(RELEASE_DOCUMENTS)
 
 pdfs: $(RELEASE_PDFS)
 
+github_issues: $(SYSTEM_CONFIG)
```

Comment:
For now, we should make this target as being "PHONY" since we have no way of knowing when it should be run or not, and thus it should just always be run.  We can probably optimize this in the future.